src/openrct2/peep/Peep.cpp

```diff
@@ -5061,6 +5093,8 @@ static bool peep_update_fixing_sub_state_9(bool firstRun, rct_peep * peep, Ride
 
 /**
  * rct2: 0x006C1368
+ * sub_state_10: applies to fixing station specific breakdowns: safety cut-out, control failure, inspection.
```

Comment:
Fix station start

Reply:
@duncanspumpkin Thanks for all these naming suggestions. Very helpful.